<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Particula</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>22</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rojo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>141</y>
         <width>45</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Velocidad</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>20</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Azul</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>52</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Color (rgb)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>117</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino en y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>47</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen en x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Verde</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen en y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="agregar_inicio_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>307</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar inicio</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino en x</string>
       </property>
      </widget>
      <widget class="QPushButton" name="agregar_final_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>339</y>
         <width>307</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar final</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mostrar_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>363</y>
         <width>307</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mostrar</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="id_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>23</y>
         <width>241</width>
         <height>18</height>
        </rect>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="origen_x_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>47</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="origen_y_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>70</y>
         <width>241</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="destino_x_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>94</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="destino_y_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>117</y>
         <width>241</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="velocidad_spinBox">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>141</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="rojo_spinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>241</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>225</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="verde_spinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="maximum">
        <number>225</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="azul_spinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>270</y>
         <width>241</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>225</number>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>311</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
